index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html class="no-js" lang="english">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Delicious - The Best Food Blog on the Web!</title>
+    <meta name="food blog" content="How to make chocolate pizza">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <link rel="stylesheet" href="css/normalize.min.css">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+    <!--[if lt IE 8]>
+      <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
+    <header>
+      <img src="images/logo.png" alt="logo" class="logo">
+      <section class="logo">
+        <h1 class="logoStyleOne" class="logoText">Delicious</h1>
+        <h1 class="logoStyleTwo" class="logoText">THE BEST FOOD BLOG ON THE WEB.<h1>
+      </section>
+      <nav>
+        <ul class="nav">
+          <li>
+            <a href="">
+              <img src="images/fb-icon.png" alt="facebook">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/twit-icon.png" alt="twitter">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/gp-icon.png" alt="google plus">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/insta-icon.png" alt="instagram">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/flic-icon.png" alt="flickr">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/pint-icon.png" alt="pintrest">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/rss-icon.png" alt="rss">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/mail-icon.png" alt="mail">
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="border"></div>
```

Comment:
This border is purely used for style. Style should be handled by CSS exclusively. Add a border to something that exists rather than creating an entirely new element.